content/webapp/utils/digital-guides.ts

```diff
@@ -94,7 +95,8 @@ export const getGuidesRedirections = (
 
   // The first stop's link is an exception, in that it should link to
   // the [exhibitionId]/[type] page, and not directly on the stop's page.
-  if (hasValidUserPreference && hasValidStopNumber) {
+  // TODO remove hasEgWorkCookie check once toggle is removed
+  if (hasEgWorkCookie && hasValidUserPreference && hasValidStopNumber) {
```

Comment:
This could've been done in the parent PR, but they'll get merged together.
It's probably low risk to have this work for everyone anyway, but it's just redirecting 404s to other 404s.